Subject:  Compile time fixes for changes to device_memory from commit
26bea849cfa1d020150e0862002d7d5463f07817 ("Cleanup: add device_texture for
images, distinct from other global memory")
Patch Author:  Orson Teodoro <orsonteodoro@hotmail.com>

Adapted from blender patches.
----
--- a/src/device/device_network.h.orig	2020-03-26 06:00:35.000000000 -0700
+++ b/src/device/device_network.h	2020-07-08 00:46:14.366062596 -0700
@@ -130,7 +130,6 @@ class RPCSend {
     archive &mem.data_type &mem.data_elements &mem.data_size;
     archive &mem.data_width &mem.data_height &mem.data_depth &mem.device_pointer;
     archive &mem.type &string(mem.name);
-    archive &mem.interpolation &mem.extension;
     archive &mem.device_pointer;
   }
 
@@ -272,7 +272,6 @@ class RPCReceive {
     *archive &mem.data_type &mem.data_elements &mem.data_size;
     *archive &mem.data_width &mem.data_height &mem.data_depth &mem.device_pointer;
     *archive &mem.type &name;
-    *archive &mem.interpolation &mem.extension;
     *archive &mem.device_pointer;
 
     mem.name = name.c_str();
